docs/source/getting-started/index.rst

```diff
@@ -68,251 +74,100 @@ You can read through the quickstart as a guide, or navigate directly to the note
+with only a single line of code for many of the most popular ML libraries in the Python ecosystem.
+
+.. figure:: ../_static/images/tutorials/introductory/autologging-intro.png
+    :width: 80%
+    :align: center
+    :alt: The basics of MLflow tracking.
+
+In this brief tutorial, you'll learn how to leverage MLflow's autologging feature to simplify your model logging activities.
 
 .. raw:: html
 
      <section>
         <article class="simple-grid">
             <div class="simple-card">
-                <a href="quickstart-1/index.html#add-mlflow-tracking-to-your-code" >
-                    <div class="header">
-                        Introduction to autologging
-                    </div>
-                    <p>
-                        Train a model and use MLflow autologging to automatically record model artifacts, metrics, and parameters
-                    </p>
-                </a>
-            </div>
-            <div class="simple-card">
-                <a href="quickstart-1/index.html#view-mlflow-runs-and-experiments" >
+                <a href="../tracking/autolog.html" >
                     <div class="header">
-                        View autologged data in the MLflow UI
+                        MLflow Autologging Quickstart
                     </div>
                     <p>
-                        See what autologging will autonomously log for you during model training with only a single line of code
-                    </p>
-                </a>
-            </div>
-            <div class="simple-card">
-                <a href="quickstart-1/index.html#load-a-model-from-a-specific-training-run-for-inference" >
-                    <div class="header">
-                        Loading a model for inference
-                    </div>
-                    <p>
-                        Load the autologged model in its native format and use it to generate predictions
+                    Get started with logging to MLflow with the high-level autologging API in a fast-paced guide 
                     </p>
                 </a>
             </div>
         </article>
     </section>
 
-.. toctree::
-    :maxdepth: 1
-    :hidden:
 
-    quickstart-1/index
 
-`15 minute Quickstart - Comparing Runs and Deploying your Best Model <quickstart-2/index.html>`_
-------------------------------------------------------------------------------------------------
+Run Comparison Basics
+^^^^^^^^^^^^^^^^^^^^^
 
-This quickstart tutorial focuses on the MLflow UI's run comparison feature, provides a brief overview of MLflow Projects, 
-and shows how to register a model. After locally serving the registered model, a brief example of preparing a model for remote deployment 
-via containerizing the model via Docker is covered. 
+This quickstart tutorial focuses on the MLflow UI's run comparison feature, provides a brief overview of `MLflow Projects <../projects.html>`_, 
```

Comment:
nit: this tutorial doesn't seem to mention about MLflow Project

Reply:
ooooh good catch. I totally forgot that we removed all of that from this tutorial. Thanks!